internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,444 @@
+package asyncapi
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/antihax/optional"
+	kafkarestv3 "github.com/confluentinc/ccloud-sdk-go-v2/kafkarest/v3"
+	srsdk "github.com/confluentinc/schema-registry-sdk-go"
+	"github.com/go-yaml/yaml"
+	"github.com/iancoleman/strcase"
+	"github.com/spf13/cobra"
+	spec2 "github.com/swaggest/go-asyncapi/spec-2.4.0"
+	yaml3 "k8s.io/apimachinery/pkg/util/yaml"
+
+	"github.com/confluentinc/cli/internal/pkg/ccloudv2"
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/kafkarest"
+	"github.com/confluentinc/cli/internal/pkg/log"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+const parseErrorMessage string = "topic is already present and overwrite flag is false. Moving to the next topic"
```

Comment:
Can we remove this second sentence? It's not part of the error and might be better suited as a log message.
```suggestion
const parseErrorMessage string = "topic is already present and overwrite flag is false"
```